Taarafo.Portal.Web/Brokers/Loggings/LoggingBroker.cs

```diff
@@ -12,7 +12,7 @@ public class LoggingBroker : ILoggingBroker
     {
         private readonly ILogger logger;
 
-        public LoggingBroker(ILogger logger) => this.logger = logger;
+        public LoggingBroker(ILogger<LoggingBroker> logger) => this.logger = logger;
```

Comment:
This change is not a part of this PR.

Reply:
Hay sorry i did revert this one and will create an issue for this if not already one created